pgactivity/views.py

```diff
@@ -242,7 +242,7 @@ def indent(text: str) -> str:
         columns = [
             [f"* Global: {render(uptime)} uptime"],
             [f"{render(total_size)} dbs size - {render(size_ev)} growth"],
-            [f"{render(si.cache_hit_ratio)} cache hit ratio"],
+            [f"{render(si.cache_hit_ratio_last_snap)} cache hit ratio"],
```

Comment:
How does this render if `None`? (Not sure that can happen, per comment above.)

Reply:
It will render as "-" because the renderer will match NoneType.

https://github.com/dalibo/pg_activity/blob/b42264982b93d9052372338ca24ac3fec6b4145f/pgactivity/views.py#L174-L177

Reply:
Ah, I didn't remember that; that's great!